userguide/tutorials/development.adoc

```diff
@@ -507,24 +375,129 @@ The steps above configure the application so that the Kill Bill logs are display
 
   <log_file_path>/killbill.out
 
-=== Setting up a Breakpoint and Remote Debugging - TODO
+=== Setting up a Breakpoint and Remote Debugging
 
 Sometimes, you may face some issues in running the application. In such cases, it is useful to set up a breakpoint and debug the application. Here is how you can do this:
 
-. Set up a breakpoint in `DefaultKillbillConfigSource.java#L118` (Press Ctrl + T and type `DefaultKillbillConfigSource`)
+. Create a new environment variable *MAVEN_OPTS* and set it to `-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n`
+
+
+. Do the following in Eclipse:
+
+.. Press Ctrl + Shift + T and locate `DefaultKillbillConfigSource`. 
+
+..  If this does not show the source code do the following:
+
+... Click *Attach Source*. 
+
+... Click on *External Location* and Navigate to your Maven local repository (`~/.m2/repository`). Select `/org/kill-bill/billing/killbill-platform-base/0.40.4/killbill-platform-base-0.40.4-sources.jar`
```

Comment:
Maybe specify that this would change depending on the file that needs to be debugged (the version might change too).

Reply:
Fixed the description to make it more general. Please let me know if you still feel that some additional information is required.